graphql-cli/src/main/java/io/ballerina/graphql/generator/ballerina/ClientGenerator.java

```diff
@@ -100,19 +103,16 @@ public static ClientGenerator getInstance() {
     /**
      * Generates the client file content.
      *
-     * @param queryDocument                     the object instance of the queries document
-     * @param queryDocumentName                 the name of the queries document
      * @param graphQLSchema                     the object instance of the GraphQL schema (SDL)
```

Comment:
Let's add the documentation comment for the new param `queryDocuments`.

Suggestion `@param queryDocuments        the list of names of the queries documents`